src/index.js

```diff
@@ -1,11 +1,16 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please 🙏🏽 don't disable linters, instead of fixing them manually, it will help you to use better practices of JS.